README.md

```diff
@@ -0,0 +1,13 @@
+# ToDo App
+
+## Installation
+
+```bash
+pip install -r requirements.txt
```

Comment:
Please add instructions about: cloning the project, creating a virtual env and run the server.